ballerina/tests/10-custom-datetime-test.bal

```diff
@@ -26,6 +26,8 @@ isolated function beforeInsertTimeFunc() returns sql:Error? {
    result = check oracledbClient->execute(`CREATE TABLE TestDateTimeTable(
        PK NUMBER GENERATED ALWAYS AS IDENTITY,
        COL_DATE  DATE,
+       COL_DATE_ONLY  DATE,
+       COL_TIME_ONLY  INTERVAL DAY(0) TO SECOND,
```

Comment:
Isn't this type `INTERVAL DAY TO SECOND ` without precision

Reply:
Since there is no Type to save the Time only data, considerable count of oracledb users use this `INTERVAL DAY(0) TO SECOND` type to save the time related data. that is why I added that to a test case to test it out.

Reply:
Intervals will handle in a separate issue https://github.com/ballerina-platform/ballerina-standard-library/issues/1763